plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -107,11 +105,40 @@ export class TaskWorker {
     });
   }
 
+  private async registerAbortable(task: TaskManager) {
+    const taskId = await task.getWorkspaceName();
+    let shouldUnsubscribe = false;
+
+    const subscription = this.options.taskBroker
+      .event$({ taskId, after: undefined })
+      .subscribe({
+        error: _ => {
+          shouldUnsubscribe = true;
```

Comment:
so could it happen that this subscription lingers forever (potential memory leak right?) if an error occurs that's not followed by a regular message?